test/mailers/previews/review_mailer_preview.rb

```diff
@@ -0,0 +1,17 @@
+class ReviewMailerPreview < ActionMailer::Preview
+  def review_needed_email
+    ReviewMailer.review_needed_email(Draft.first)
+  end
```

Comment:
1. Yeah, this was working on my local, not sure what's making it not work on yours.
2. That sounds like a good idea. It looks like you can specify a custom path for them to live at. What would be a good location for them?

Reply:
Hmm - that's a good question. 🤔 I don't have strong opinions about it but it looks like the other option in the docs is `lib/mailer_previews` ... we could put them in there. Or maybe `rspec/mailers/`? I don't have strong opinions.

Reply:
Ok, I've gone ahead and moved them to `lib/mailer_previews` since they aren't specifically `rspec`-related. Do you a list like the following at `localhost:3000/rails/mailers`?

<img width="1041" alt="screen shot 2018-12-16 at 2 51 35 pm" src="https://user-images.githubusercontent.com/2397451/50059928-1dbe7080-0142-11e9-9179-4d235f162468.png">